index.html

```diff
@@ -45,7 +45,7 @@ <h1 id="title">Survay Form</h1>
     Feel free to delete this message once you have read it.
     Good luck and Happy Coding!
 
-    - The freeCodeCamp Team
+    - The freeCodeCamp Team :)

```

Comment:
-I suggest you to make more descriptive commits.
-Kindly, you should not modify the comment section.

Reply:
- [ ] :red_circle: [REQUIRED] This comment is unclear. Please make sure when you comment you also give more details about it. By this I mean, tell the project's owner why we need a more descriptive commit and how a descriptive commit looks like. You could say for example
 **I suggest you make a more descriptive commit message. Commit messages should be meaningful and should tell what changes are inside. For example, if you add a title to your index.html the respective commit message can be something  like  `Add page title  to index.html `**

Reply:
- [ ] :red_circle: [REQUIRED] This comment is unclear. Please make sure when you comment you also give more details about it. By this I mean, tell the project's owner why we need a more descriptive commit and how a descriptive commit looks like. You could say for example
 **I suggest you make a more descriptive commit message. Commit messages should be meaningful and should tell what changes are inside. For example, if you add a title to your index.html the respective commit message can be something  like  `Add page title  to index.html `**

Reply:
- [ ] I suggest you make a more descriptive commit message. [This article could help you with that.](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/) Commit messages should be meaningful and should tell what changes are inside. For example, if you add a title to your index.html the respective commit message can be something like Add page title to index.html